client/src/components/TabularViewComponent/TabularViewComponent.scss

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+@import "../../theme/theme.scss";
+
+.ms-DetailsList {
+    overflow-x: hidden;
```

Comment:
Do we use 4 space tabs in other CSS files in this project? I can't remember. Let's just be consistent though.

Also, perhaps it would be useful to add a component specific prefix to this CSS overrides? E.g. `.tabular-view` (and add this to the enclosing div in the component as a class).

Reply:
Done